packages/gdl-frontend/components/HomePage/index.js

```diff
@@ -91,12 +99,39 @@ type Props = {|
     { featuredContent: Array<FeaturedContent> }
   >,
   languageCode: string,
-  featuredContent: FeaturedContent,
+  featuredContent: Array<FeaturedContent>,
   categories: Array<Category>,
   category: Category
 |};
 
-class HomePage extends React.Component<Props> {
+type State = { index: number };
+
+class HomePage extends React.Component<Props, State> {
+  state = { index: 0 };
```

Comment:
Jeg vil foreslå å trekke ut denne staten vekk fra `HomePage` ettersom den egentlig bare tilhører `FeaturedCaroussel`. Det betyr jeg tenker det er lurt å trekke ut koden nede ut i en egen komponent også.

Største grunnen er at ved state endring i `HomePage` så rerendres hele denne komponenten. Det fører til flere uønskede rerendringer enn mann måtte ønske og kan potensielt føre til bugs. 
I tillegg til at den ene merge konflikten du får kommer av at jeg tar en rerendring sjekk med `shouldComponentDidUpdate` for å bestemme når denne komponenten skal rendres.